pontoon/api/schema.py

```diff
@@ -0,0 +1,124 @@
+import graphene
+from graphene_django import DjangoObjectType
+from graphene_django.debug import DjangoDebug
+
+from .util import get_fields
+
+from ..base.models import (
+    Project as ProjectModel,
+    Locale as LocaleModel,
+    ProjectLocale as ProjectLocaleModel
+)
+
+
+class StringStats(graphene.AbstractType):
+    missing_strings = graphene.Int()
+
+    @graphene.resolve_only_args
+    def resolve_missing_strings(self):
+        return (
+            self.total_strings - self.translated_strings -
+            self.approved_strings - self.fuzzy_strings
```

Comment:
I think you missed this question in the previous review (it's my fault, it was mixed in with another comment). Is ``rejected`` not missing here? What is ``translated``? How is it different from ``approved``? cc @mathjazz

Reply:
translated is suggested here. sorry. :)

Reply:
I used the definition I found here: https://github.com/mozilla/pontoon/blob/dd266843219e14eb41a7d871a63d7750f0386857/pontoon/base/templates/widgets/progress_chart.html#L35

Does this look right?

Reply:
It does look right.

Added a comment in https://bugzilla.mozilla.org/show_bug.cgi?id=1377969#c1.

Reply:
@mathjazz why do we not need to account for rejected strings here?

Reply:
Stats only keep track of active translations, which can be either:
* Translated (`approved_strings`)
* Suggested (`translated_strings`) - either unreviewed or rejected
* Fuzzy (`fuzzy_strings`)
* Missing (`resolve_missing_strings()`)

So translated + suggested + fuzzy + missing = total.

In [bug 1377969](https://bugzilla.mozilla.org/show_bug.cgi?id=1377969) we'll get rid of the Suggested and treat them as missing. So translated + fuzzy + missing = total will be true.

We'll add unreviewed suggestions as a separate count. We could also add rejected and missing counts and store them in the model.

Reply:
Thanks for that explanation!